types/index-spec.ts

```diff
@@ -30,17 +28,19 @@ const optionalConfigurationFunction = function() {
 		jasmine.addMatchers({
 			toBeTheAnswerToTheUltimateQuestionOfLifeTheUniverseAndEverything: function (util, customEqualityTesters) {
 				return {
-					compare: function (actual) {
-						let result = {};
+					compare: function (actual: unknown) {
+						let result: {pass: boolean, message?: string} = {pass: true};
```

Comment:
I took this from the readme, so you likely need to update the readme too.

Reply:
It does actually work without this update typescript just isn't smart enough to know that `pass` is set below this line. This change is only needed to please the typescript compiler.